src/main/resources/application-local.yml

```
@@ -0,0 +1,6 @@
+spring:
+  config:
+    activate:
+      on-profile: local
+    import:
+      - classpath:/secret/application-local.yml
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)